order.go

```diff
@@ -0,0 +1,108 @@
+package apiutil
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+var (
+	// OrderASC defines the ascending order keyword
+	OrderASC = "asc"
+	// OrderDESC defines the descending order keyword
+	OrderDESC = "desc"
+)
+
+// OrderBy represents ordering information
+type OrderBy struct {
+	OrderItems []Order
+}
+
+// Order represents ordering information for a single key
+type Order struct {
+	Key       string
+	Direction string
+}
+
+// NewOrderBy constructs a basic order by struct
+func NewOrderBy() OrderBy {
+	return OrderBy{
+		OrderItems: []Order{},
+	}
+}
```

Comment:
Generally I prefer constructors appear just after the type declaration (this is where godoc puts them). In this case I think the constructor should be dropped if we switch to a type declaration on a slice, and direct users to use slice allocations to declare OrderBys:

```
apiutil.OrderBy{}
```